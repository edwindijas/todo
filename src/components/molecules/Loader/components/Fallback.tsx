import ProgressActivityIcon from '@/components/atoms/Icon/ProgressActivity'

const Fallback = (): JSX.Element => (
  <div className='w-full h-full flex items-center justify-center'>
    <ProgressActivityIcon className='w-14 h-14 animate-spin' />
  </div>
)
export default Fallback