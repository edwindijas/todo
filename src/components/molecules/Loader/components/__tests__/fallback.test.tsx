import { render } from '@testing-library/react'
import Fallback from '../Fallback'

it ('renders snapshot tests correctly', () => {
  const {container} = render(<Fallback />)
  expect(container).toMatchSnapshot()
})  