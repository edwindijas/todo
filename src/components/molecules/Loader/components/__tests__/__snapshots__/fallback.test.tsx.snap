// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot tests correctly 1`] = `
<div>
  <div
    class="w-full h-full flex items-center justify-center"
  >
    <iconmock
      class="w-14 h-14 animate-spin w-full h-full"
      data-testid="component-atom-icon"
    />
  </div>
</div>
`;
